# SPDX-FileCopyrightText: 2024 Nextcloud GmbH and Nextcloud contributors
# SPDX-License-Identifier: AGPL-3.0-or-later
diff --git a/node_modules/vue-tabs-component/src/components/Tabs.vue b/node_modules/vue-tabs-component/src/components/Tabs.vue
index 2a87c6c..8625dfb 100644
--- a/node_modules/vue-tabs-component/src/components/Tabs.vue
+++ b/node_modules/vue-tabs-component/src/components/Tabs.vue
@@ -26,7 +26,7 @@
 </template>
 
 <script>
-    import expiringStorage from '../expiringStorage';
+    import expiringStorage from '../expiringStorage.js';
 
     export default {
         props: {
diff --git a/node_modules/vue-tabs-component/src/index.js b/node_modules/vue-tabs-component/src/index.js
index 8c38dd4..ee45ac1 100644
--- a/node_modules/vue-tabs-component/src/index.js
+++ b/node_modules/vue-tabs-component/src/index.js
@@ -1,5 +1,5 @@
-import Tab from './components/Tab';
-import Tabs from './components/Tabs';
+import Tab from './components/Tab.vue';
+import Tabs from './components/Tabs.vue';
 
 export default {
     install(Vue) {
